meson/dependencies/openMP/meson.build

```diff
@@ -1,7 +1,16 @@
 omp = dependency('openmp', required: false)
 
 if not omp.found()
-    error('\n\nYour compiler does not support openMP. Please reconfigure with -DWITH_OMP=false\n')
+    # it might be your meson is a bit old
```

Comment:
I don't understand why this modification is here.. did you rebase on the current v3.0?

Reply:
In my local and remote this file looks different, I don't know why you see it differently....
https://github.com/ElisaBortoli/geotop/blob/with_openmp/meson/dependencies/openMP/meson.build